html/login/index.php

```diff
@@ -10,7 +11,20 @@
 	if(	array_key_exists('user',$_POST)
 	&&	array_key_exists('passwd',$_POST) ){
 		$login_result = $db->login($_POST['user'],$_POST['passwd']);
-		if(!$login_result){
+		if($login_result){
+            if(array_key_exists('redirect_to',$_POST) && array_key_exists('signature',$_POST)) {
+	            $crypt = new crypt();
+	            $uri = $_POST['redirect_to'];
+	            $signature = $_POST['signature'];
+                
+                if ($crypt->verify($uri, $signature)) {
+                    header('Location: ' . $uri, true, 303);
+                    die();
+                }
+            }
+            header('Location: /');
```

Comment:
redirect to /login/ on fail, just like current behaviour. It doubles as the logout page - and maybe account page in the future. We'll see.